### PR TITLE
CVSL-259: Update packages to fix security audit job

### PR DESCRIPTION
**Pull request by timharrison-moj**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -2983,11 +2983,6 @@
         "@types/node": "*"
       }
     },
-    "@types/validator": {
```
Comment (smcveigh941) [inline package-lock.json:2986]: Should this one be removed?

File: package-lock.json [hunk 2]
```diff
@@ -2983,11 +2983,6 @@
...
-      "version": "13.6.3",
-      "resolved": "https://registry.npmjs.org/@types/validator/-/validator-13.6.3.tgz",
-      "integrity": "sha512-fWG42pMJOL4jKsDDZZREnXLjc3UE0R8LOJfARWYg6U966rxDT7TYejYzLnUF5cvSObGg34nd0+H2wHHU5Omdfw=="
```
Comment (smcveigh941) [inline package-lock.json:2989]: Should there be `package.json` changes to go along with this?

Comment (timharrison-moj) [reply package-lock.json:2989]: Security audit was failing so I just ran `npm audit fix` , then `npm run security_audit` and its passing.  But yes, this should have updated package.json too, but didn't. Hmm.. I'll update class-validator to 0.13.2 as well.

Comment (timharrison-moj) [reply package-lock.json:2989]: It was saying anything above ^0.13.1 - but I'll update anyway.

File: package-lock.json (hunk 1 above)
Comment (timharrison-moj) [reply package-lock.json:2986]: It has, hasn't it?

Comment (timharrison-moj) [reply package-lock.json:2986]: Ah, you mean *should* it have been!

Comment (timharrison-moj) [reply package-lock.json:2986]: Assuming later version of class-validator didn't need it - but I'll check.

File: package-lock.json (hunk 2 above)
Comment (timharrison-moj) [reply package-lock.json:2989]: Updated now. Everything else looks ok - all tests passing and no audit issues.

**Review by smcveigh941 — APPROVED** [review]
LGTM

**Pull request merged**